samples/meetings-sidepanel/csharp/README.md

```diff
@@ -0,0 +1,50 @@
+
+This sample illustrates how to implement [Side Panel](https://docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/create-apps-for-teams-meetings?view=msteams-client-js-latest&tabs=dotnet#notificationsignal-api) In-Meeting Experience.
+  
+### User interactions(Meeting Organizer)
+- **Add New Agenda Item** - Gives provision to add new Agenda point.
+- **Add** - Adds the agenda from Textinput to the SidePanel agenda list.
+- **Publish Agenda** - Sends the agenda list to the meeting chat.
+
+## Prerequisites
+
+### Tools
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+- [Ngrok](https://ngrok.com/download) (Only for devbox testing) Latest (any other tunneling software can also be used)
+  ```bash
+  # run ngrok locally
+  ngrok http -host-header=localhost 3978
+  ```
+- [SignalR](https://docs.microsoft.com/en-us/aspnet/signalr/overview/getting-started/tutorial-getting-started-with-signalr-and-mvc) To update agenda in Real-Time
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+1. Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+2. If you are using Visual Studio
+- Launch Visual Studio
+- File -> Open -> Project/Solution
+- Navigate to ```samples\meetings-sidepanel\csharp``` folder
+- Select ```SidePanel.sln``` file
+3. Run ngrok - point to port 3978
+   ```ngrok http -host-header=rewrite 3978```
+4. Create a new Bot by following steps mentioned in [Build a bot](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/what-are-bots?view=msteams-client-js-latest#build--a-bot-for-teams-with-the-microsoft-bot-framework) documentation.
+5. Go to appsettings.json and add ```MicrosoftAppId```, ```MicrosoftAppPassword``` and ```BaseUrl``` information.
+6. Update the manifest.json file with ```Microsoft-App-ID``` and ```BaseUrl``` value.
+7. Run your app, either from Visual Studio with ```F5``` or using ```dotnet run``` in the appropriate folder.
+8. [Install the App in Teams Meeting](https://docs.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-apps-in-meetings?view=msteams-client-js-latest#meeting-lifecycle-scenarios)
+
+## Interacting with the app in Teams Meeting
+Interact with SidePanel by clicking on the App icon present on the top menu beside the "more actions" during a meeting.
+1. Organizer need to ping the bot by mentioning ```@``` in the meeting chat in order to gain the Organizer privilage of the app to Add and Publish the Agenda points.
+2. Once the app is clicked, sidepanel appears with the default agenda list. Only organizer gets the feasibility to add new agenda points to the list using "Add New Agenda Item" button.
+3. On click of "Add" button, agenda point will be added to the agenda list.
+4. On click of "Publish Agenda", the agenda list will be sent to the meeting chat.
```

Comment:
Add images reference which are added in description to readme file.  Copied references here: 

![image](https://user-images.githubusercontent.com/50989436/111726512-1f31f280-888f-11eb-8e96-cce2f8a8d456.png)

![image](https://user-images.githubusercontent.com/50989436/111726569-3bce2a80-888f-11eb-8ba6-1c662b2939da.png)

![image](https://user-images.githubusercontent.com/50989436/111726656-5accbc80-888f-11eb-94e3-af1bc18bd500.png)